docs/technical_reference/unit_models/gac.rst

```diff
@@ -90,10 +91,10 @@ Sets
 
    "Time", ":math:`t`", "[0]"
    "Phases", ":math:`p`", "['Liq']"
-   "Components", ":math:`j`", "['H2O', 'Solute', 'Background solutes/ions]*"
+   "Components", ":math:`j`", "['H2O', 'adsorbed solute', 'background solutes]*"
```

Comment:
```suggestion
   "Components", ":math:`j`", "['H2O', 'target_species', 'background solutes]*"
```
Why not just call this target species and explain below that this represents the adsorbed solute?